src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -80,24 +79,24 @@ static Cookie()
         }
 #endif
 
-        // These PreserveDependency attributes are a workaround for https://github.com/dotnet/runtime/issues/19348.
```

Comment:
Not for you to fix here, but reading this message, it sounds like we should move these to a “library build” xml descriptor file instead. That way we don’t need to hack around this with ILKeepDepsAttributes=false. I’ll open an issue for this on Monday.